controllers/cluster_controller.go

```diff
@@ -615,9 +625,112 @@ func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&hwameistoroperatorv1alpha1.Cluster{}).
 		Owns(&appsv1.DaemonSet{}).
 		Owns(&appsv1.Deployment{}).
+		Owns(&batchv1.Job{}).Watches(&source.Kind{Type: &batchv1.Job{}},
+		&handler.EnqueueRequestForObject{},
+		builder.WithPredicates(
+			predicate.Funcs{
+				UpdateFunc: r.handleDrbdJobUpdatePredicate,
+				DeleteFunc: r.handleDrbdJobDeletePredicate,
+			},
+		)).
 		Complete(r)
 }
 
+func (r *ClusterReconciler) handleDrbdJobUpdatePredicate(e event.UpdateEvent) bool {
+	newJob := e.ObjectNew.(*batchv1.Job)
+	r.handleDrbdJobUpdate(r.Client, newJob)
+	return true
+}
+
+func (r *ClusterReconciler) handleDrbdJobDeletePredicate(e event.DeleteEvent) bool {
+	job := e.Object.(*batchv1.Job)
+	r.handleDrbdJobDelete(r.Client, job)
+	return false
+}
+
+func (r *ClusterReconciler) handleDrbdJobUpdate(cli client.Client, job *batchv1.Job) error {
+	if !isJobComplete(job) {
+		return nil
+	}
+	log.Infof("Job %s has successfully completed.", job.Name)
+	var nodeList corev1.NodeList
+	if err := cli.List(context.TODO(), &nodeList); err != nil {
+		log.Errorf("List nodes err: %v", err)
+		return err
+	}
+	var jobList batchv1.JobList
+	if err := cli.List(context.TODO(), &jobList, client.InNamespace("hwameistor")); err != nil {
+		log.Errorf("List jobs err: %v", err)
+		return err
+	}
+	count := 0
+	for _, item := range jobList.Items {
+		if strings.HasPrefix(item.Name, "drbd-adapter") && isJobComplete(&item) {
+			count++
+		}
+	}
+
+	if count >= len(nodeList.Items) {
```

Comment:
this can be simpified by judging status of each job, update to `Completed` when all jobs are in `Completed` status, otherwise do nothing(or log some incompleted jobs)